src/navigator-card-body.jsx

```diff
@@ -375,7 +375,7 @@ function Item({ file, selected, setSelected, onDoubleClickNavigate, isGrid }) {
         if (ev.detail > 1) {
             onDoubleClickNavigate(file);
         } else {
-            if (!ev.ctrlKey || selected === path[path.length - 1]) {
```

Comment:
How does this help? Selected is still passed to `Item`.

Reply:
Ah, I suppose this was the check for if where `selected` == the current directory which was just super confusing.